_includes/releases/v22.1/v22.1.0-beta.3.md

```diff
@@ -0,0 +1,78 @@
+- `EXPLAIN (DDL)`, when invoked on statements supported by the declarative schema changer, prints a plan of what the schema changer will do. This can be useful for anticipating the complexity of a schema change (e.g., anything involving backfill or validation operations might be slow to run) and for troubleshooting. `EXPLAIN (DDL, VERBOSE)` produces a more detailed plan. [#79780][#79780]
+
+<h3 id="v22-1-0-beta-3-operational-changes">Operational changes</h3>
+
+- Added a new time-series metric, `storage.marked-for-compaction-files`, for the count of files marked for compaction. This is useful for monitoring storage-level background migrations. [#79370][#79370]
+- [Changefeed](../v22.1/change-data-capture-overview.html) creation and failure event logs are now emitted to the [`TELEMETRY` logging channel](../v22.1/eventlog.html#telemetry-events). [#79749][#79749]
+
+<h3 id="v22-1-0-beta-3-command-line-changes">Command-line changes</h3>
+
+- Introduced a new `ttllogger` [workload](../v22.1/cockroach-workload.html) which creates a TTL table emulating a "log" with rows expiring after the duration specified in the `--ttl` flag. [#	][#79482]
+
+<h3 id="v22-1-0-beta-3-db-console-changes">DB Console changes</h3>
+
+- Filtering by column has been added to the [Hot Ranges page](../v22.1/ui-hot-ranges-page.html). [#79647][#79647]
+
+<h3 id="v22-1-0-beta-3-bug-fixes">Bug fixes</h3>
+
+- Previously, privileges for restored tables were being generated incorrectly without taking into consideration their parent schema's default privilege descriptor. This is now fixed. [#79534][#79534]
+- Fixed a bug that caused an internal error when the inner expression of a column access expression evaluated to `NULL`. For example, evaluation of the expression `(CASE WHEN b THEN ((ROW(1) AS a)) ELSE NULL END).a` would error when `b` is `false`. This bug was present since v19.1 or earlier. [#79529][#79529]
+- Fixed a bug that caused an error when accessing a named column of a labeled tuple. The bug only occurred when an expression could produce one of several different tuples. For example, `(CASE WHEN true THEN (ROW(1) AS a) ELSE (ROW(2) AS a) END).a` would fail to evaluate. This bug was present since v22.1.0. Although present in previous versions, it was impossible to encounter due to limitations that prevented using tuples in this way. [#79529][#79529]
+- Previously, queries reading from an index or primary key on `FLOAT` or `REAL` columns `DESC` would read `-0` for every `+0` value stored in the index. This has been fixed to correctly read `+0` for `+0` and `-0` for `-0`. [#79533][#79533]
+- Separate tenants now use unique values for the internal field `ClusterID`. Previously, separate tenants that were hosted on the same storage cluster would be assigned the same `ClusterID`. [#79742][#79742]
```

Comment:
```suggestion
```

Drop because we are no longer including multi-tenant architecture features in release notes.